tools/gen_ort_dockerfile.py

```diff
@@ -153,9 +154,11 @@ def dockerfile_for_linux(output_file):
     && . /etc/os-release \\
     && echo "deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ $UBUNTU_CODENAME main" | tee /etc/apt/sources.list.d/kitware.list >/dev/null \\
     && apt-get update -q=2 \\
-    && apt-get install -y --no-install-recommends cmake=3.27.7* cmake-data=3.27.7* \\
+    && apt-get install -y --no-install-recommends cmake=3.28.* cmake-data=3.28.* \\
```

Comment:
Version 3.28 has a bug with CUDA Toolkit, it was fixed in latest versions.
https://gitlab.kitware.com/cmake/cmake/-/issues/25484